bot.py

```
@@ -1,7 +1,31 @@
-from app import run
+# === FORCE RENDER PORT BINDING ON BOOT ===
+class DummyHandler(BaseHTTPRequestHandler):
+    def do_GET(self):
+        self.send_response(200)
+        self.end_headers()
+        self.wfile.write(b"Bot is alive!")
+
+def run_dummy_server():
+    port = int(os.environ.get("PORT", 10000))
+    try:
+        server = HTTPServer(("0.0.0.0", port), DummyHandler)
```

Ruff (0.15.18)

[error] Possible binding to all interfaces

(S104)